main.pm

```diff
@@ -857,6 +857,14 @@ else {
     }
     elsif (get_var("BOOT_HDD_IMAGE")) {
         loadtest "boot/boot_to_desktop.pm";
+        if (get_var("ISCSI_SERVER")) {
+            set_var('INSTALLONLY', 1);
```

Comment:
We have other variables that follow other variables - without them showing in the job_settings. I don't see the problem. But I would indeed factor this out of the load_ functions and put them in front of the save_vars to make this variable dependency more clearer